README.asciidoc

```diff
@@ -2,6 +2,98 @@
 
 = Team "Väike My" in PiCR 2022/23
 
+== Members
+
+. Rasmus Saame
+. Michelle Lukken
+. Leonid Zinatullin
+
+== Links
+
+Fusion 360 link: https://a360.co/3HeGZ8V
+
+== Documentation
+
+=== Programming
+
+See software/README.asciidoc
+
+=== Electronics
+
+See Electronics branch electronics/README.asciidoc
```

Comment:
Checked it. I think it can be done as a separate PR but some comments while this is not done:
* Missing images of the board
* Missing analysis of the design, good, bad, what to improve